index.html

```diff
@@ -77,7 +77,165 @@ <h3 class="username">Karma</h3>
         </div>
     </div>
     <div class="content">
-      <h3>content</h3>
+      <div class="content-header">
+        <h2 class="content-header-item">Home</h2>
+        <i class="fa fa-certificate" title="Top Tweets"></i>
+      </div>
+      <div class="what-happening-box">
+        <div class="tweet-box">
+          <a href="#" class="user-card">
+            <img src="images/avatar.png" alt="avatar" class="user-avatar"/>
+          </a>
+          <div class="text-privacy-container">
+            <textarea name="tweet" class="tweet-field" cols="40" rows="1" placeholder="What's happening?" required></textarea>
+            <div class="hint">
+              <a href="#"  class="hint-message">
+                  <i class="fa fa-globe"></i> Everyone can reply
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="options">
+          <div class="icons">
```

Comment:
this naming is bad, very broad, use a very specific naming here

Reply:
icons: didn't use this class , removed